doc/developers/advanced_installation.rst

```diff
@@ -86,6 +89,9 @@ feature, code or documentation improvement).
    .. prompt:: bash $
 
      pip install --verbose --no-build-isolation --editable .
+     
+   Be sure to be in the folder of the scikit-learn repository before run the above command,
+   otherwise you will face errors.
```

Comment:
It may be. Actually, after I first cloned the repository I went directly to the next step and didn't notice that there was also to do the cd after the clone (that it's also obvious after all). I think that a friendly reminder here it's well accepted :)

Reply:
In that case, I prefer have another step after cloning that says "navigate to the scikit-learn directory". Would that help you?

Reply:
Yes, I think that will be clearer and less prone to missing the passage